### PR TITLE
Fix: Fixed crash with properties window

### LINKED ISSUE #12057: Bug: Property window crashes randomly
### Description

Files suddenly crashes when repeatedly opening and closing property windows or moving sections.
The timing of the crash is random, and the same steps may or may not result in a crash.

### Steps To Reproduce

1. Open and close property windows or move sections repeatedly.
2. Files crashes.

### Requirements

Do not crash.

### Files Version

2.4.61.0

### Windows Version

Windows 11 22H2

### Log file

No error log found. AccessViolationException seems to occur.
スクリーンショット 2023-04-18 121930

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12057 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/App.xaml.cs [hunk 1]
```diff
@@ -61,6 +61,7 @@ public partial class App : Application
 		public static string OutputPath { get; set; }
 		public static CommandBarFlyout? LastOpenedFlyout { get; set; }
 		public static StorageHistoryWrapper HistoryWrapper = new StorageHistoryWrapper();
+		public static bool isMainWindowClosed { get; private set; } = false;
```
Comment (yaira2) [inline src/Files.App/App.xaml.cs:64]: This should probably be in a data model

Comment (hishitetsu) [reply src/Files.App/App.xaml.cs:64]: You mean AppModel?

Comment (yaira2) [reply src/Files.App/App.xaml.cs:64]: Yeah

Comment (hishitetsu) [reply src/Files.App/App.xaml.cs]: Done.

File: src/Files.App/Helpers/FilePropertiesHelpers.cs [hunk 2]
```diff
@@ -97,17 +101,21 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
...
+			}
+
+			propertiesWindow.IsMinimizable = false;
+			propertiesWindow.IsMaximizable = false;
+			propertiesWindow.MinWidth = 460;
+			propertiesWindow.MinHeight = 550;
+			propertiesWindow.Width = 800;
+			propertiesWindow.Height = 550;
+			propertiesWindow.Content = frame;
+			propertiesWindow.SystemBackdrop = new MicaBackdrop();
```
Comment (QuaintMako) [inline src/Files.App/Helpers/FilePropertiesHelpers.cs:118]: It was initially initialized as `MicaSystemBackdrop`. Is the change wanted?

Comment (hishitetsu) [reply src/Files.App/Helpers/FilePropertiesHelpers.cs:118]: Yes, this is necessary. I found WinUIEx's `MicaSystemBackdrop` isn't applied to cached windows. So I use WASDK's `MicaBackdrop`.

Comment (yaira2) [reply src/Files.App/Helpers/FilePropertiesHelpers.cs:118]: @hez2010 this might be useful for the work you're doing.

**Review by QuaintMako — APPROVED** [review]
LGTM, I cannot reproduce the crash with those changes.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**